include/gatekeeper_cps.h

```diff
@@ -39,6 +39,18 @@ struct cps_config {
 	 */
 	int               debug;
 
+	/* Number of times to attempt bring a KNI interface up or down. */
+	uint8_t           num_attempts_kni_link_set;
+
+	/* Maximum number of updates for LPM table to serve at once. */
+	uint16_t          max_cps_route_updates;
```

Comment:
Same as above -- could just be `unsigned int` unless there's a specific reason to limit it to the size of a 16-bit number.